service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class FurtherDirections {
+    private final String directionsNeeded;
```

Comment:
I know you started doing the JsonGetters / Setters could we model this as boolean and use the similar fashion to model it as boolean? :)

Reply:
do something like if directionsNeeded = yes, set, if no, set as empty string? definitely possible

Reply:
is it better practice to do optional logic on serialisation?

Reply:
@akomisarek